app/screens/main/Main.js

```diff
@@ -88,13 +87,18 @@ class Main extends Component<Props, State> {
             {routes.main.map((route) => (
               <Route key={route.path} path={route.path} component={route.component} />
             ))}
-            <Redirect to="/main/wallet" />
           </Switch>
         </InnerWrapper>
       </Wrapper>
     );
   }
 
+  componentDidMount() {
```

Comment:
move this to constructor

Reply:
This does not achieve the same effect. In order to set the index properly, the component needs to be already mounted.

Reply:
You need to use location object that comes with component's props and not location on window object that might be not reflect the last changes yet

Reply:
Got it.